spec/javascripts/title_module_spec.js

```diff
@@ -0,0 +1,31 @@
+var editor; var module;
+
+describe("TitleModule", function() {
+  beforeAll(function() {
+    fixture = loadFixtures('index.html');
+
+    editor = new PL.Editor({
+      textarea: $('.ple-textarea')[0]
+    });
+
+    module = new PL.TitleModule(editor, {});
+  });
+
+  it("value, valid, key", function() {
```

Comment:
Can we write properly here what this test is actually doing so that other contributors can easily figure out, thanks 😄